.nova/Configuration.json

```diff
@@ -0,0 +1,3 @@
+{
+  "workspace.color" : 9
```

Comment:
don't commit this

Reply:
@rianfowler not sure how this file even got added to the commit since it should have been ignored by git. But I'll submit a PR to clean it up.